layers/+tools/lsp/funcs.el

```diff
@@ -97,7 +97,8 @@
    ((configuration-layer/package-usedp 'helm)
     (spacemacs/set-leader-keys-for-minor-mode 'lsp-mode
       (concat prefix-char "s") #'helm-lsp-workspace-symbol
-      (concat prefix-char "S") #'helm-lsp-global-workspace-symbol))
+      (concat prefix-char "S") #'helm-lsp-global-workspace-symbol)
+    (spacemacs/set-leader-keys "pE" #'helm-lsp-diagnostics))
```

Comment:
Do we have a similar function for ivy?

Reply:
ATM no, but I guess soon we will have it implemented by lsp-mode team member that users ivy.